src/sdk/pynni/nni/common.py

```diff
@@ -52,6 +67,20 @@ def write(self, s):
             self.file.flush()
         return len(s)
 
+def get_log_level():
+    if env_args.log_level:
+        level_number = logLevelNameMap.get(env_args.log_level)
+        if level_number:
+            if level_number >= DEBUG:
+                log_level = logging.DEBUG
+            elif level_number >= INFO:
+                log_level = logging.INFO
+            elif level_number >= WARNING:
+                log_level = logging.WARNING
+            elif level_number >= ERROR:
```

Comment:
Suggest to simplify this section, just put logging.xxx into the map.

Reply:
Good suggestion, updated.